tests/test_modeling_tf_common.py

```diff
@@ -753,7 +753,7 @@ def test_resize_token_embeddings(self):
 
     def test_lm_head_model_random_no_beam_search_generate(self):
         config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
-        input_ids = inputs_dict["input_ids"] if "input_ids" in inputs_dict else inputs_dict["inputs"]
+        input_ids = inputs_dict["input_ids"]  # if "input_ids" in inputs_dict else inputs_dict["inputs"]
```

Comment:
I think we can clean up this comment and remove it thanks to all your hard work!